app.py

```diff
@@ -1,9 +1,41 @@
+import uuid
 from flask import render_template
 
 from server import create_app, run_app, db, socketio
 from server.models import AuthType, Users, Projects, Sources
 
 
+
+
+def new_google_user(profile):
+    user_name = profile["name"]
+    email = profile["email"]
+    profile_picture = profile["imageUrl"]
+    auth_type = AuthType.GOOGLE
+    user_id = uuid.uuid4()
+    db.session.add(Users(user_id, user_name,auth_type,email,profile_picture));
+    db.session.commit();
+
+def new_facebook_user(profile):
+    user_name = profile["name"]
+    email = "sample email"#profile["email"]
```

Comment:
Since this field can have a null value, we can pass in None instead of a placeholder string. This applies to `new_microsoft_user` as well.